HappyAnding/HappyAnding/Model/SectionType.swift

```diff
@@ -106,4 +106,87 @@ enum SectionType {
                 )
         }
     }
+    
+    func fetchTitleImage() -> some View {
+        switch self {
+        case .recent:
+            return Image("new")
+                .resizable()
+                .scaledToFit()
+                .frame(width: 19)
+                .foregroundStyle(
+                    Color(hexString: "E4C139"),
+                    Color(hexString: "E4C139")
+                )
+                .shadow(color: Color(hexString: "E5C239").opacity(0.36),
+                            radius: 8,
+                            x: 0,
+                            y: 0)
+        case .download:
+            return Image("ranked")
+                .resizable()
+                .scaledToFit()
+                .frame(width: 19)
+                .foregroundStyle(
+                    Color(hexString: "404040", opacity: 0.48),
```

Comment:
```suggestion
                    Color("charcoalGray").opacity: 0.48),
```
모든 #404040은 Color("charcoalGray")입니다

Reply:
반영했습니다.